src/UserRepository.js

```diff
@@ -20,6 +20,13 @@ class UserRepository {
     return Math.round((sum / data.sleepToDate.length));
   }
 
+  getTheWeekOf(date) {
```

Comment:
It looks like this takes in date as an argument, but dates as an argument on line 22 of Activity.js. Am I misunderstanding something?